boxes/__init__.py

```diff
@@ -198,7 +198,7 @@ def html(self, name, default, translate):
             ("""<option value="%s"%s>%s</option>""" %
              (e, ' selected="selected"' if e == default else "",
               translate("%s %s" % (e, self.names.get(e, "")))) for e in self.edges))
-        return """<select name="%s" size="1">\n%s</select>\n""" % (name, options) #here
+        return """<select name="%s" id="%s" aria-labeledby="%s %s" size="1">\n%s</select>\n""" % (name,  name, name+"_id", name+"_description", options)
```

Comment:
this adds and id and aria-labeled by that references the id, name, and description from the boxesserver file this allows for screen readers to announce the label to the left of the input and the description to the right of the input for the combo box